ms2query/create_sqlite_database.py

```diff
@@ -14,15 +14,18 @@
 import pandas as pd
 import numpy as np
 import sqlite3
+import os
+import gc
 from matchms import Spectrum
 from ms2query.app_helpers import load_pickled_file
 
 
 def make_sqlfile_wrapper(sqlite_file_name: str,
-                         npy_file_location: str,
-                         pickled_file_name: str,
-                         csv_file_with_inchikey_order: str
-                         ):
+                         tanimoto_scores_pickled_dataframe_file: str,
+                         list_of_spectra: List[Spectrum],
```

Comment:
```suggestion
                         spectrums: List[Spectrum],
```

Reply:
That would be consistent with matchms, spec2vec, and ms2deepscore.

Reply:
Good point, I think a lot of ms2query functions could benefit from having a good look at consistent/clear naming. 
In the case of spectrums, I tried to consistently use spectra, so if I change it here, I think I should do the same for all other functions.
A reason why I preferred using list_of_spectra over spectrums is, that it can be confusing to see the difference between spectrum and spectrums, which makes it a bit confusing in some functions. 

So shall I change spectra to spectrums in all functions so it is consistent with matchms and spec2vec or is consistency within ms2query enough?